.gitignore

```diff
@@ -58,3 +58,6 @@ target/
 
 # DS_Store
 *.DS_Store
+
+# pycharm
```

Comment:
@noahzaozao If you think it's necessary to have this in .gitignore (which I think is ok), then we should propagate to all our repos